src/context_store/storage/prisma.py

```
@@ -171,31 +119,27 @@ async def run(self) -> None:
+            pass
+        await self._apply_migration(file_path)
 
     async def _get_applied_migrations(self) -> set[str]:
         try:
             rows = await self._client.query_raw("SELECT version FROM schema_migrations")
-        except RawQueryError as e:
-            error_str = str(e).lower()
-            if "does not exist" in error_str or "42p01" in error_str:
-                return set()
-            raise
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)